dashboard/views.py

```diff
@@ -23,3 +24,13 @@ def get_context_data(self, **kwargs):
         context['domains'] = domains
         context['push_apps'] = push_apps
         return context
+
+
+class VapidValidation(TemplateView):
+    template_name = 'dashboard/vapid_validation.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(VapidValidation, self).get_context_data(**kwargs)
+        context['vapid_validation_form'] = VapidValidationForm()
+        context['jws_token'] = generate_jws_key_token()
```

Comment:
Shouldn't need this in the view. We will generate the jws key token one time when the push application is created and save it to the database. When we show it to the user, we will show them the value out of the database.